src/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/GovUk.Education.ExploreEducationStatistics.Public.Data.Api.csproj

```diff
@@ -5,9 +5,17 @@
         <Nullable>enable</Nullable>
         <ImplicitUsings>enable</ImplicitUsings>
         <RootNamespace>GovUk.Education.ExploreEducationStatistics.Public.Data.Api</RootNamespace>
+
+        <IncludeOpenAPIAnalyzers>true</IncludeOpenAPIAnalyzers>
+        <GenerateDocumentationFile>true</GenerateDocumentationFile>
+        
+        <!-- Disable warnings for undocumented methods -->
+        <NoWarn>1591</NoWarn> 
```

Comment:
Do we want this? If it applies to only controller actions then maybe we want the warning if we have missed the documentation?

Reply:
We need this otherwise the build logs start getting spammed with warnings like the following for undocumented methods and classes:

```
/GovUk.Education.ExploreEducationStatistics.Public.Data.Api/YourClass.cs(11,14): warning CS1591: Missing XML comment for publicly visible type or member 'YourMethod'
```